dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -359,23 +359,9 @@ class RubricsControllerTest < ActionController::TestCase
     assert_equal 0, json_response.length
   end
 
-  test "returns forbidden when getting aggregate status if ai experiment isn't enabled" do
```

Comment:
please keep this test, and change it to test that the api returns forbidden if the teacher is not verified.